src/sourmash/sourmash_args.py

```diff
@@ -393,14 +393,27 @@ def load_pathlist_from_file(filename):
     try:
         with open(filename, 'rt') as fp:
             file_list = [ x.rstrip('\r\n') for x in fp ]
-
-        if not os.path.exists(file_list[0]):
-            raise ValueError("first element of list-of-files does not exist")
+        if len(file_list) == 0:
+            raise ValueError("pathlist is empty")
+        badFormat = 0
+        duplicate = []
+        for i in range(len(file_list)):
+            if not os.path.exists(file_list[i]):
+                badFormat = badFormat + 1
+            if file_list[i] not in duplicate:
+                duplicate.append(file_list[i])
+        if badFormat > 0:
+            raise ValueError("pathlist contains a badly formatted file")
+        if len(duplicate) != len(file_list):
```

Comment:
see [this comment](https://github.com/dib-lab/sourmash/pull/1469#issuecomment-830353364). I don't think we need to warn about duplicates, we should just remove them (by making `file_list` a set).

Reply:
So basically I created a set called `duplicate` and I added each item in `file_list` to it. Since it is a set, it will not add any duplicate items. Is this not how I should do it?

Reply:
err, here `duplicate` is a list, is it not?

Reply:
Oh I think I messed it up! Will fix it, thanks! Just to clarify, should I do the same thing I'm currently doing after making `duplicate` a set?

Reply:
what a great question! submit something that passes the tests and let me know when you want me to review it ;)